pkg/scheduler/framework/session_plugins.go

```diff
@@ -626,6 +626,32 @@ func (ssn *Session) TaskOrderFn(l, r interface{}) bool {
 	return helpers.CompareTask(lv, rv)
 }
 
+// NodesWherePreemptionMightHelp filter out those node that has unscheduleandunre
+func (ssn *Session) NodesWherePreemptionMightHelp(task *api.TaskInfo) []*api.NodeInfo {
+	fitErrors, ok1 := ssn.Jobs[task.Job]
```

Comment:
it seems `NodesWherePreemptionMightHel` is a public function for actions. Do not place this func in session_plugin.go. session_plugin.go offers a set of interfaces for plugins to implement. e.g. JobReady(), JobOrder() etc.

Reply:
It's reasonable. Done.